### ISSUE TITLE
Identifiers

### ISSUE THREAD

**Issue by KathiSchleidt** [DataCoveEU/INSPIRE_Coverage#3]
Going through the IR Requirements, I've found an interesting one:

> (1) If elevation data is updated based on new source data, the updated objects shall receive a new external object identifier.

Pertaining to versioning, there is a (non-binding) statement that:

> A change of version should occur only when the elevation data is reprocessed using the same source data, which is the typical case when errors or thematic inconsistencies has been detected in the data and therefore it .

**Comment by pebau** [comment]
that Req does not solve the problem for a component coverage.
BTW, is an identifier the carrier of referencing? In other words, does new identifier mean disconnected from the coverage? This would render updates impossible, but produce zombie subcoverages instead.

**Comment by KathiSchleidt** [comment]
Actually, to my understanding this identifier requirement pertains to the INSPIRE ID, so think we're safe!
I had the following insight falling asleep last night (when I do my best work ;) ):
The INSPIRE ID gets shifted to the metadata element together with the rest of the INSPIRE Extensions, thus, as long as the metadata element pertaining to the updated coverage subset is replaced and provided with a new INSPIRE ID, everything should be OK

Question is how hard it is to switch individual metadata records from the list contained within a coverage. Worst case the provider of the update would have to:

- download the list of existing metadata records 
- replace the md records for the updated subsets
- adjust footprints of md records if required
- reload the new list of metadata records

**Comment by pebau** [comment]
yes, that is what I wanted to express in our communication: updating a coverage by adding a subcoverage may require extensive updates of existing subcoverages in case of overlaps. Not sure that is what member states will enjoy.

BTW, according to our experience updates by far are not just for correcting errors, but a common step when (i) stepwise adding files to a coverage or (ii) updating on ortho image because some new highway route has to be reflected or (iii) any other housekeeping. Again according to experience agencies usually do not keep the previous state thy want to have the same object appear to their customers. A new version of the subcoverage, who cares? And should that include new versions for updated footprints of coverages overlapped by the new patch?

As I said: aggregated coverages considered bad. Only ESA wants them, but all suffer (service providers as well as implementers).
